EvolvSwiftSDK/EvolvSwiftSDKTests/Common-Tests/EvolvStoreNewTests.swift

```diff
@@ -101,8 +101,8 @@ class EvolvStoreNewTests: XCTestCase {
         let evolvStore = initializeEvolvStore(with: context)
         let activeKeys = evolvStore.getActiveKeys()
         
-        XCTAssert(activeKeys.contains("next"))
-        XCTAssertEqual(activeKeys.count, 1)
+        XCTAssert(activeKeys.contains("next.layout"))
```

Comment:
Can you keep in the test that activeKeys still contains 'next'?

Reply:
Sure, we should keep both "next.layout" and "next" check

Reply:
yes please - worth checking all the active keys explicitly